packages/web/src/app/[lng]/(auth)/memos/_components/ExtensionInstallCheckDialog/index.tsx

```
@@ -3,14 +3,14 @@
 import { URL } from '@extension/shared/constants';
 import { checkLocalStorageTrue, setLocalStorageTrue } from '@extension/shared/modules/local-storage';
 import {
-  Button,
-  Dialog,
-  DialogContent,
-  DialogDescription,
-  DialogFooter,
-  DialogHeader,
-  DialogTitle,
-} from '@src/components/ui';
+    Button,
```

ESLint

[error] Delete `··`

(prettier/prettier)